module/entityAssessors/helper.js

```diff
@@ -831,4 +836,45 @@ module.exports = class EntityAssessorHelper {
         })
     }
 
+    /**
+     * Create entity assessor data.
+     * @method
+     * @name create
+     * @param {String} programId - program id.
+     * @param {String} solutionId - solution id.
+     * @param {String} entityType - entity type.
+     * @param {String} entityTypeId - entity type id.
+     * @param {Array}  entities - entities 
+     * @param {Object} userDetails - logged in user details. 
+     * @returns {Object} created Entity assessor data.
+     */
+
+    static create(programId,solutionId,entityType,entityTypeId, entities, userDetails) {
```

Comment:
@aman-tunerlabs - Use a common query to create an entity assessor as already done in this file.